spec/features/workflow/edit_withdrawal_spec.rb

```diff
@@ -33,14 +35,20 @@ def then_i_can_see_the_existing_public_explanation
   end
 
   def and_i_can_edit_the_public_explanation
-    new_explanation = "Another explanation"
-    converted_explanation = GovspeakDocument.new(new_explanation, @edition).payload_html
+    @new_explanation = "Another explanation"
+    converted_explanation = GovspeakDocument.new(@new_explanation, @edition).payload_html
     body = { type: "withdrawal", explanation: converted_explanation, locale: @edition.locale }
     stub_publishing_api_unpublish(@edition.content_id, body: body)
 
-    fill_in "public_explanation", with: new_explanation
-    click_on "Withdraw document"
+    fill_in "public_explanation", with: @new_explanation
+    click_on "Update explanation"
+  end
+
+  def then_i_can_see_the_updated_explanation
+    expect(page).to have_content(@new_explanation)
+  end
 
-    expect(@edition.reload.status.details.public_explanation).to eq(new_explanation)
+  def and_i_can_see_the_history_entry
```

Comment:
Minor one: I think we usually fold this step into the previous one; as most features involve a timeline entry, it seems better to avoid the extra step everywhere when reading the scenario. Or maybe we should make it explicit everywhere?

Reply:
Ah I wasn't aware of that convention, will do as you suggest.